src/pages/AppDetails/AppInfo/AppDescription.tsx

```diff
@@ -36,103 +48,94 @@ class AppDescription extends React.Component<AppDescriptionProps, AppDescription
-    /*
-      Not sure if we need a common icon per Workload instead of an icon per type of Workload
-     */
-    const iconName = WorkloadIcon;
-    const iconType = 'pf';
-    const heading = (
-      <div className="ServiceList-Heading">
-        <div className="ServiceList-Title">
-          <div className="component-label">
-            Workload{' '}
-            {!workload.istioSidecar && (
-              <MissingSidecar namespace={namespace} style={{ marginLeft: '10px' }} tooltip={true} text={''} />
-            )}
-          </div>
-          <Link to={this.workloadLink(namespace, workload.workloadName)}>{workload.workloadName}</Link>
-        </div>
-      </div>
-    );
-    const content = (
-      <ListViewItem
-        leftContent={<ListViewIcon type={iconType} name={iconName} />}
-        key={`AppWorkload_${workload.workloadName}`}
-        heading={heading}
-      />
+    return (
+      <ListItem key={`AppWorkload_${workload.workloadName}`}>
+        <Link to={this.workloadLink(namespace, workload.workloadName)}>{workload.workloadName}</Link>
+        {!workload.istioSidecar && (
+          <MissingSidecar namespace={namespace} style={{ marginLeft: '10px' }} tooltip={true} text={''} />
+        )}
+      </ListItem>
     );
-    return content;
   }
 
   renderServiceItem(namespace: string, _appName: string, serviceName: string) {
-    const iconName = 'service';
-    const iconType = 'pf';
-    const heading = (
-      <div className="ServiceList-Heading">
-        <div className="ServiceList-Title">
-          <div className="component-label">Service</div>
-          <Link to={this.serviceLink(namespace, serviceName)}>{serviceName}</Link>
-        </div>
-      </div>
+    return (
+      <ListItem key={`AppService_${serviceName}`}>
+        <Link to={this.serviceLink(namespace, serviceName)}>{serviceName}</Link>
+      </ListItem>
     );
-    const content = (
-      <ListViewItem
-        leftContent={<ListViewIcon type={iconType} name={iconName} />}
-        key={`AppService_${serviceName}`}
-        heading={heading}
-      />
-    );
-    return content;
   }
 
   renderEmptyItem(type: string) {
     const message = 'No ' + type + ' found for this app.';
-    return <ListViewItem description={message} />;
+    return <DataListCell> {message} </DataListCell>;
   }
 
   workloadList() {
     const ns = this.props.app.namespace.name;
     const workloads = this.props.app.workloads;
-    return workloads.length > 0
-      ? workloads.map(wkd => this.renderWorkloadItem(ns, wkd))
-      : this.renderEmptyItem('workloads');
+    const workloadList =
+      workloads.length > 0 ? workloads.map(wkd => this.renderWorkloadItem(ns, wkd)) : this.renderEmptyItem('workloads');
+
+    return [
+      <DataListCell key="workload-icon" isIcon={true}>
+        <Badge>W</Badge>
+      </DataListCell>,
+      <DataListCell key="workload-list" className="resourceList">
+        <h3>Workloads</h3>
+        <List>{workloadList}</List>
```

Comment:
What do you think about https://patternfly-react.surge.sh/patternfly-4/components/text/ instead of h3 ?

``typescript
 <Text component={TextVariants.h3}>Third level</Text>
```